python/tk_multi_publish2/dialog.py

```diff
@@ -1225,6 +1326,42 @@ def _publish_again_clicked(self):
         # reset the validation flag
         self._validation_run = False
 
+    def _get_inherited_description_item(self, item):
```

Comment:
What do you think or renaming this to `_find_inherited_description`? I had a hard time understanding the code originally as `get` often implies something that is ready to use when actually here we're searching for one.

Reply:
yeah thats a fair point actually, I'll rename it.